### PR TITLE
fixed issue where old scr colormap file would not load opacity

### PR DESCRIPTION
**Pull request by doutriaux1**
@aashish24 @sankhesh @chaosphere2112 just found a tiny bug. Please merge.

### CODE REVIEW

File: Packages/vcs/Lib/colormap.py [hunk 1]
```diff
@@ -33,7 +33,7 @@ def process_src(nm, code):
     for i in range(255):
         val = numbers[i * 3:i * 3 + 3]
         if val != ():
-            d[i] = list(val)
+            d[i] = list(val)+[100]
```
Comment (aashish24) [inline Packages/vcs/Lib/colormap.py]: @doutriaux1 can you exaplain bit more what this code is doing?

Comment (doutriaux1) [inline Packages/vcs/Lib/colormap.py]: old colormaps only had r,g,b values so when loading back in it only stores 3 values, then later on when using getcolor we unpack it into r,g,b,a causing a failure. Voila.

Comment (sankhesh) [inline Packages/vcs/Lib/colormap.py]: Not Flake8 compliant.

Need spaces around binary operator "+".

Comment (doutriaux1) [inline Packages/vcs/Lib/colormap.py]: yet flake8 passes, bwwwaaaaaa I beat flake8!!!!!

**Comment by aashish24** [comment]
LGTM :+1:

**Pull request merged**